src/dummy.rs

```diff
@@ -66,8 +66,17 @@ pub fn dummy_value(val_ty: ValType) -> Val {
         ValType::F32 => Val::F32(0),
         ValType::F64 => Val::F64(0),
         ValType::V128 => Val::V128(0.into()),
-        ValType::ExternRef => Val::ExternRef(None),
-        ValType::FuncRef => Val::FuncRef(None),
+        ValType::Ref(ref_type) => {
+            if ref_type.matches(&RefType::EXTERNREF) {
```

Comment:
Can you check whether `ref_type.is_nullable()` and return an error here ("cannot create a dummy value for a non-nullable reference type") if so? This will require making a few of these `dummy_foo` functions return `Result<Foo>`s.

Reply:
was this the intended change?